### PR TITLE
Compatibility changes for V8 7.4 support

### PR DESCRIPTION
**Pull request by ignisf**

### CODE REVIEW

File: ext/mini_racer_extension/mini_racer_extension.cc [hunk 1]
```diff
@@ -152,7 +152,7 @@ static VALUE rb_mJSON;
 static VALUE rb_cFailedV8Conversion;
 static VALUE rb_cDateTime = Qnil;
 
-static Platform* current_platform = NULL;
+static std::unique_ptr<Platform> current_platform = NULL;
```
Comment (ignisf) [inline ext/mini_racer_extension/mini_racer_extension.cc]: @SamSaffron, check this out. The new platform API mandates the use of an `sstd::unique_ptr`.

Comment (SamSaffron) [reply ext/mini_racer_extension/mini_racer_extension.cc]: oh interesting!

**Comment by SamSaffron** [comment]
OK lets try this out! CI is freaking out about bundler :( will get it sorted

**Comment by ignisf** [comment]
They should be releasing a new version of bundler that no longer has this issue.

**Pull request merged**